src/github.com/getlantern/flashlight/util/http.go

```diff
@@ -2,21 +2,138 @@ package util
 
 import (
 	"crypto/tls"
+	"errors"
 	"fmt"
 	"net"
 	"net/http"
 	"net/url"
+	"sync/atomic"
 	"time"
 
+	"github.com/getlantern/fronted"
 	"github.com/getlantern/golog"
 	"github.com/getlantern/keyman"
 	"github.com/getlantern/waitforserver"
 )
 
+const (
+	defaultAddr = "127.0.0.1:8787"
```

Comment:
I came across this when running different versions of Lantern instance on same VPS to measure load time. I supplied different `-addr=` options to each instance, and lots of `connection refused` error reported during config update and geolookup. I redirected output to `/dev/null` to prevent those errors from blowing up the whole disk.

I agree that regular Lantern client will hardly change the port, so it's probably ok.